tests/src/ExistingSite/FetchClientUrlTest.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drupal\Tests\collabora_online\ExistingSite;
+
+use Drupal\collabora_online\Cool\CoolRequest;
+use weitzman\DrupalTestTraits\ExistingSiteBase;
+
+/**
+ * Tests requests to Collabora from PHP.
+ */
+class FetchClientUrlTest extends ExistingSiteBase {
```

Comment:
I might drop this test, it was just an attempt.

Reply:
Yeah..
I was not sure if this functionality fails which other test would properly detect it.
I am sure we will see failures but not clean ones.

We could also leave this test as a placeholder.
If you look at the history, the first version covers the old `CoolRequest` class.
Once we have a dedicated test for the service we can remove this one.